src/test/ui/reify-intrinsic.stderr

```diff
@@ -8,9 +8,9 @@ LL |     let _: unsafe extern "rust-intrinsic" fn(isize) -> usize = std::mem::tr
    |                                                                help: use parentheses to call this function: `std::mem::transmute(...)`
    |
    = note: expected type `unsafe extern "rust-intrinsic" fn(isize) -> usize`
-              found type `unsafe extern "rust-intrinsic" fn(_) -> _ {std::intrinsics::transmute::<_, _>}`
+              found type `unsafe extern "rust-intrinsic" fn(_) -> _ {std::mem::transmute::<_, _>}`
```

Comment:
Uh oh... I wonder if you'll run into the same troubles as in https://github.com/rust-lang/rust/pull/63948#issuecomment-533873566.

Reply:
?

Reply:
There seems to be some problem with platform dependent diagnostic output which caused #63948 to fail somehow and it's the same test file as the one here.

Reply:
Weird.